examples/supply-chain-app/package.json

```diff
@@ -48,7 +48,8 @@
   "homepage": "https://github.com/hyperledger/cactus#readme",
   "dependencies": {
     "@hyperledger/cactus-example-supply-chain-backend": "file:../cactus-example-supply-chain-backend",
-    "@hyperledger/cactus-example-supply-chain-frontend": "file:../cactus-example-supply-chain-frontend"
+    "@hyperledger/cactus-example-supply-chain-frontend": "file:../cactus-example-supply-chain-frontend",
+    "dotenv": "^8.2.0"
```

Comment:
@jordigiam Please lock the version down to an exact one (so no caret, wildcard, etc).

Reply:
Done! Thanks for the advice!